src/main/resources/db/migration/V5__alter_user_table.sql

```diff
@@ -0,0 +1,6 @@
+UPDATE users
+SET password = 'default_password'
```

Comment:
Since we are using a hash password in the database, we can set the password using a hash value.

also instead of making the password field not null. just set a default password.

Reply:
instead of set not null
it can be SET DEFAULT 'hash-password value'

Reply:
Setting the password of users whose password field are null solves the bug and then afterward the constraints is added so no password is null